### ISSUE TITLE
Script used to generate the compressed files?

### ISSUE THREAD

**Issue by SandroGrzicic** [me-no-dev/esp8266fcgui#2]
First of all, great work!

Can you share the script you used to generate the compressed files? This would allow using the latest Configurators if needed. Thanks.

**Comment by me-no-dev** [comment]
yes I will :) I need to clean some stuff up. Images are manually inlined though, but those will likely not change.

**Comment by me-no-dev** [comment]
There you go :) https://github.com/me-no-dev/esp8266fcgui/commit/534a7677b5fb22f1fa9ae78ccdadeb5e09bfd21b

**Comment by kmitchel** [comment]
Ty.

**Comment by SandroGrzicic** [comment]
Thanks!

**Comment by SandroGrzicic** [comment]
BTW - I'm not sure it's the best idea to include the actual source files since they change very often, the script should just return an error indicating where to get the sources instead if they aren't downloaded yet, I think.

**Comment by me-no-dev** [comment]
code is modified, so I think it's a good idea if someone wants to update it :) if you get the source as it is from the respectfull places, it will not upload and work

**Comment by SandroGrzicic** [comment]
Ah so the betaflight configurator included in this repo is not the normal betaflight configurator? can you summarise the changes if we want to use a later version? :)

I suppose the BlHeli configurator would then need the same modifications applied?

**Comment by me-no-dev** [comment]
All images are inlined in the css and js (only a few are left as is). All flashing stuff has been removed and the Serial driver has been replaced with WebSocket client. The method of translation has also changes (original uses chrome api).
The rest is the same and can be diffed against newer git versions and updated accordingly :)
